docs/_index.md

```diff
@@ -105,6 +105,10 @@ Viam supports a wide variety of systems, including:
   <div class="prev" style="display: block">‹</div>
   <div class="next" style="display: block">›</div>
 </div>
+<br>
+
+Viam is designed to be flexible and extensible, and to save you time from prototype to production.
```

Comment:
This strikes me as potentially too sales-y for the docs page.

Reply:
Your probably right. Trying for that elusive padding:  `Explore the following resources to jump right in:` just seems like too little between sections.